tests/python/gaia-ui-tests/gaiatest/apps/settings/regions/device_info.py

```diff
@@ -42,12 +70,58 @@ def model(self):
     def software(self):
         return self.marionette.find_element(*self._software_locator).text
 
+    def tap_your_rights(self):
+        Wait(self.marionette).until(expected.element_displayed(
+            self.marionette.find_element(*self._your_rights_locator)))
+        self.marionette.find_element(*self._your_rights_locator).tap()
+        Wait(self.marionette).until(expected.element_displayed(
+            self.marionette.find_element(*self._rights_page_locator)))
+
+    def tap_your_privacy(self):
+        Wait(self.marionette).until(expected.element_displayed(
+            self.marionette.find_element(*self._your_privacy_locator)))
+        self.marionette.find_element(*self._your_privacy_locator).tap()
+        Wait(self.marionette).until(expected.element_displayed(
+            self.marionette.find_element(*self._privacy_page_header_locator)))
+
+    def tap_legal_info(self):
+        Wait(self.marionette).until(expected.element_displayed(
+            self.marionette.find_element(*self._legal_info_locator)))
+
+        self.marionette.find_element(*self._legal_info_locator).tap()
+        Wait(self.marionette).until(expected.element_displayed(
+            self.marionette.find_element(*self._open_source_notices_locator)))
+
+    def tap_open_source_notices(self):
+        Wait(self.marionette).until(expected.element_displayed(
+            self.marionette.find_element(*self._open_source_notices_locator)))
+
+        self.marionette.find_element(*self._open_source_notices_locator).tap()
+        Wait(self.marionette).until(expected.element_displayed(
+            self.marionette.find_element(*self._notice_text_locator)))
+        time.sleep(3)  # loads the HTML file, needs to wait until it completes loading the page
```

Comment:
Perhaps you could crawl in the page, then look for an element to be visible to prevent this sleep() call, but I guess this is fine too. It might give problem on our slow Flame, though, no idea.